tests/quaternion.rs

```diff
@@ -113,6 +113,36 @@ mod from {
     }
 }
 
+mod arc {
+    use cgmath::*;
+
+    fn test(src: Vector3<f32>, dst: Vector3<f32>) {
+        let q = Quaternion::from_arc(src, dst, None);
+        let v = q.rotate_vector(src);
+        assert_approx_eq!(v, dst);
+    }
```

Comment:
The idea was to actually re-use it for more tests, at some point.

> On Jul 5, 2016, at 19:17, Brendan Zabarauskas <notifications@github.com> wrote:
> 
> In tests/quaternion.rs:
> 
> > @@ -113,6 +113,36 @@ mod from {
> >      }
> >  }
> >  
> > +mod arc {
> > +    use cgmath::*;
> > +
> > +    fn test(src: Vector3<f32>, dst: Vector3<f32>) {
> > +        let q = Quaternion::from_arc(src, dst, None);
> > +        let v = q.rotate_vector(src);
> > +        assert_approx_eq!(v, dst);
> > +    }
> Inline as in put the code in test_opposite, rather than in a separate function ^_^
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>